birdhouse/optional-components/all-public-access/config/geoserver/permissions.cfg

```diff
@@ -0,0 +1,79 @@
+permissions:
+ - service: geoserver
+   resource: /
+   type: route
```

Comment:
No problem. I don't think I understood how to set this properly with the new geoserver. Do you mind pointing me to the documentation or an example please

Reply:
The `resource` field is needed mostly when applying a permission on a child resource under the service. Since the permission is applied directly on the service such that it applies for all its children, the `resource` shouldn't be needed. 
The config might be completely valid, I just don't remember if Magpie code handled the `/` correctly to resolve as the `service`. If it does, it's good to leave the config as is.